api/types/schema.go

```diff
@@ -0,0 +1,7 @@
+package apitypes
+
+import "github.com/puppetlabs/wash/plugin"
+
+// EntrySchema describes an entry's schema, which
+// is what's returned by the /fs/schema endpoint.
+type EntrySchema = plugin.EntrySchema
```

Comment:
I kind of want to adopt this pattern for `plugin.EntryAttributes` + the actions. It'd be nice if the cmd's did not depend on the `plugin` package, but I can see the redundancy behind that (esp. w.r.t the actions) so am happy to remove this if that's preferred.

Reply:
I'm ok with that pattern. I think I've used it sparingly elsewhere.

Reply:
Ok. I'll put up a small cleanup PR after this to adopt this pattern where it makes sense.